### ISSUE TITLE
Error: Unable to load plugin "sqip-plugin-primitive". Try installing it via: npm install sqip-plugin-primitive

### ISSUE THREAD

**Issue by rrutsche** [axe312ger/sqip#221]
Hi there,
first of all - thanks for this package. We are using it with joy since 2 years. 
However, unfortunately it's not working on our Mac M1 machines. As the sharp dependency had to be upgraded we also upgraded sqip and the plugins to the current alpha version.
But when calling the package with the following config, we get the error 
```bash
error Error: Unable to load plugin "sqip-plugin-primitive". Try installing it via:
 npm install sqip-plugin-primitive
```

The way we call it: 
```javascript
const squipResult = await sqip({
    input: fileName,
    plugins: [
        {
            name: 'sqip-plugin-primitive',
            options: { numberOfPrimitives: 32, mode: 1 }
         },
         'sqip-plugin-svgo'
     ])
  }).catch(error => {
    console.log("error", error);
  });
``` 

These are the installed sqip related packages:
```
    "sharp": "0.30.2",
    "sqip": "1.0.0-alpha.36",
    "sqip-plugin-blur": "1.0.0-alpha.37",
    "sqip-plugin-data-uri": "1.0.0-alpha.37",
    "sqip-plugin-pixels": "1.0.0-alpha.36",
    "sqip-plugin-primitive": "1.0.0-alpha.38",
    "sqip-plugin-svgo": "1.0.0-alpha.38",
```

Did anyone else have this issue after upgrading? Are we missing something?
Thanks in advance, richard

**Comment by axe312ger** [comment]
Did you install die canary versions? I think so, otherwise you won't have the alphas :/ 

Can you check if the node_modules/sqip-plugin-primitive/package.json exists and the "main" entry points to a valid file?

**Comment by axe312ger** [comment]
@rrutsche I updated a few dependencies and rereleased. This release works fine for me on my M1.

**Comment by rrutsche** [comment]
@axe312ger Thanks for the reply and your quick release. Unfortunately it did not help.

To answer your first question: Yes there is a `node_modules/sqip-plugin-primitive/package.json` with a valid main entry point. But I think I found the issue. It's due to node version v12.22.3 we are using. Package `'fs/promises'` is not available in this version and the plugin fails to load.

I found some comments about it, pretty weird: `'fs/promises'` was being introduced in Node 10 but then removed in later patches for Node 10 and Node 12. Works fine in Node 14. Node.js documentation was incorrectly listing this as supported in Node 12 (https://github.com/nodejs/node/issues/35740).

So I guess we have to tell our hoster, that we need a node version bump.
